### DISCUSSION TITLE
Blocked Account

### DISCUSSION THREAD

**Discussion by Branco95** [vercel/community#4545] [Account Issue]
### Question

Tried several times to contact vercel via registration@vercel.com but they dont answer me. I don't know why, but the first time i try to log in with my github account in vercel, it says that the user account is blocked. Any help, please? It is very important...Thank you

**Comment by amyegan** [comment]
Sorry you're having trouble with your account. Tickets are generally processed by the order in which they are received, so it can take some time for the team to respond to all of the cases that were opened before they reach yours. If you let me know the case number, I can check on it for you.

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Reply by Branco95** [reply]
Here is my case number : 00166303 

Thank you

**Reply by Branco95** [reply]
I already emailed them a few moths ago but didn't get an answer, and my acocunt is still blocked in vercel...

**Comment by MikeRock51** [comment]
I have the same issue for a while now. What do I need to do?

**Reply by Branco95** [reply]
I am still waiting for an answer
